src/server/deployments/transformers/deployments-to-entity-row.js

```diff
@@ -21,16 +18,16 @@ function deploymentsToEntityRow(deployedService) {
     },
     {
       kind: 'tag',
-      value: statusText,
-      classes: provideDeploymentStatusClassname(statusText)
+      value: deployedService.status,
+      classes: provideDeploymentStatusClassname(deployedService.status)
```

Comment:
`provideDeploymentStatusClassname` this can just be removed and we use the status vanilla from the API?